lib/_debugger.js

```diff
@@ -126,7 +126,8 @@ Protocol.prototype.execute = function(d) {
       break;
 
     default:
-      throw new Error('Unknown state');
+      const errors = require('internal/errors');
```

Comment:
When there are only one or two cases in a file, I did it like this so the
require is only called in error conditions
On May 9, 2016 5:27 PM, "Rod Vagg" <notifications@github.com> wrote:

> In lib/_debugger.js
> <https://github.com/nodejs/node/pull/6573#discussion_r62595987>:
>
> > @@ -126,7 +126,8 @@ Protocol.prototype.execute = function(d) {
> >        break;
> >
> >      default:
> > -      throw new Error('Unknown state');
> > +      const errors = require('internal/errors');
>
> shouldn't requires be at the top of the file?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/6573/files/56d34d36a841cee62adb657548b90bd27ebce870#r62595987>
>